zandahealth/HubSpot.NET
# PD-9612 Custom Objects API methods implementation

## Description
This PR adds new methods and fixes the existing methods to work with custom objects.

## Purpose
- [x] Custom Objects API methods implemented

## Steps to test
To execute the Custom Object integration tests you need to setup a `Machine` object via a script. See more in the README file of the IntegrationTests project. Then run `HubSpotCompanyApiAsyncIntegrationTests`.

Partially resolves: [PD-9612](https://powerdiary.atlassian.net/browse/PD-9612)


[PD-9612]: https://powerdiary.atlassian.net/browse/PD-9612?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
